LDAR_Sim/testing/end_to_end_testing/test_suite/V4-simple_repairable_emissions/params/M_OGI.yaml

```diff
@@ -0,0 +1,15 @@
+parameter_level: methods
+version: "4.0"
+method_name: OGI
+deployment_type: mobile
+measurement_scale: component
+sensor:
+  type: OGI_camera_zim
```

Comment:
No MDL provided here but it is for the follow-up, why?

Reply:
To test that both ways work. 
With OGI_camera_zim, if the MDL isn't provided, there's a default that is used.